### DISCUSSION TITLE
How can I revert a manual payment entry

### DISCUSSION THREAD

**Discussion by FlohGro-dev** [pretix/pretix#2654] [Q&A]
I accidently marked an order as (partly) paid.
Is it somehow possible to delete the payment entry from the history?

**Comment by raphaelm** [comment]
You can create a refund to balance it out.

There is no deleting things in pretix, as pretix is an audit-proof system.

**Comment by FlohGro-dev** [comment]
My bad.. how can I revert a created refund?
Problem is:
I have two events. A customer cancelled one event and booked at the other event. 
i created a payment at the second event with the amount already paid at the first. 
accidently the paid money was refunded for the first event anyways. 
now I have the created payment in event two but can’t revert it 😂
